### ISSUE TITLE
Non-standard "automationFramework" capability being sent by Appium remote driver

### ISSUE THREAD

**Issue by DmytroLapshyn** [katalon-studio/katalon-studio#187]
#### Operating System: (for example: Windows 10 or OSX Sierra): 
Ubuntu 18.04.3 LTS

#### Katalon Studio Version: 
6.3.0 rc7

#### Katalon Studio logs: 
Please see [the comment from Maria Zaitseva below](https://github.com/katalon-studio/katalon-studio/issues/187#issuecomment-521934597).

#### Environment (for Mobile testing):

* Latest versions of Appium and Selenium Grid Hub

* Mobile platform/version under test: Android 8

* Android SDK simulator for Nexus 6

#### Steps to reproduce:

1. Install Selenium Grid Hub and register an Appium node with the hub
2. Configure a custom remote execution in Katalon Studio for mobile remote (i.e. Appium driver)
3. Create a test suite collection with a test suite matched with the custom remote execution configured at step 2
4. Try running the test suite collection

#### Expected Behavior:

Katalon Studio correctly initiates a session with Selenium Grid Hub and executes the test case.

#### Actual Behavior:

Even though a remote desired capability called "automationFramework" was never configured for the custom remote execution, the Katalon Studio's Appium client adds such a desired capability anyway. This, in turn, causes a failure in the Selenium Grid Hub's desired capabilities validator:

`java.lang.IllegalArgumentException: Illegal key values seen in w3c capabilities: [automationFramework]`

(an additional open question here is why there's no fallback to MJSONWP now that Katalon Studio "thinks" it talks to a remote Appium instance)

If we however use the recommended workaround for Kobiton (see the screenshot below), all works as expected.

Also JFYI, configuring a custom capability matcher in Selenium Grid Hub does NOT help.

#### Screenshots / Videos: (please attach screenshots or videos if necessary to reproduce the issue)

Using the approach recommended for Kobiton seems to work around the issue:
![Selection_054](https://user-images.githubusercontent.com/16682230/62950657-278f9500-bdf1-11e9-8067-a7b837432ae9.png)

**Comment by maria-zaitseva** [comment]
From log file:

```
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'mobile-264ub', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-55-generic', java.version: '1.8.0_222'
Driver info: driver.version: SwipeableAndroidDriver
	at com.kms.katalon.composer.components.impl.dialogs.ProgressMonitorDialogWithThread.runAndWait(ProgressMonitorDialogWithThread.java:49)
	at com.kms.katalon.composer.mobile.objectspy.components.RemoteAppComposite$1.run(RemoteAppComposite.java:111)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)
Caused by: org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Unable to parse remote response: <html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 500 Server Error</title>
</head>
<body><h2>HTTP ERROR 500</h2>
<p>Problem accessing /wd/hub/session. Reason:
<pre>    Server Error</pre></p><h3>Caused by:</h3><pre>java.io.IOException: org.openqa.grid.common.exception.GridException: Cannot extract a capabilities from the request: {
  &quot;desiredCapabilities&quot;: {
    &quot;app&quot;: &quot;com.android.calculator2&quot;,
    &quot;appActivity&quot;: &quot;com.android.calculator2.Calculator&quot;,
    &quot;appPackage&quot;: &quot;com.android.calculator2&quot;,
    &quot;platformVersion&quot;: &quot;8.0&quot;,
    &quot;automationFramework&quot;: &quot;Katalon&quot;,
    &quot;platformName&quot;: &quot;Android&quot;,
    &quot;deviceName&quot;: &quot;Nexus 6&quot;
  },
  &quot;capabilities&quot;: {
    &quot;firstMatch&quot;: [
      {
        &quot;appium:app&quot;: &quot;com.android.calculator2&quot;,
        &quot;appium:appActivity&quot;: &quot;com.android.calculator2.Calculator&quot;,
        &quot;appium:appPackage&quot;: &quot;com.android.calculator2&quot;,
        &quot;automationFramework&quot;: &quot;Katalon&quot;,
        &quot;appium:deviceName&quot;: &quot;Nexus 6&quot;,
        &quot;platformName&quot;: &quot;android&quot;,
        &quot;appium:platformVersion&quot;: &quot;8.0&quot;
      }
    ]
  }
}
	at org.openqa.grid.web.servlet.DriverServlet.process(DriverServlet.java:138)
	at org.openqa.grid.web.servlet.DriverServlet.doPost(DriverServlet.java:69)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.seleniumhq.jetty9.servlet.ServletHolder.handle(ServletHolder.java:865)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doHandle(ServletHandler.java:535)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:146)
	at org.seleniumhq.jetty9.security.SecurityHandler.handle(SecurityHandler.java:548)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doScope(SessionHandler.java:1564)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.Server.handle(Server.java:503)
	at org.seleniumhq.jetty9.server.HttpChannel.handle(HttpChannel.java:364)
	at org.seleniumhq.jetty9.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.seleniumhq.jetty9.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.seleniumhq.jetty9.io.FillInterest.fillable(FillInterest.java:103)
	at org.seleniumhq.jetty9.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.seleniumhq.jetty9.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.openqa.grid.common.exception.GridException: Cannot extract a capabilities from the request: {
  &quot;desiredCapabilities&quot;: {
    &quot;app&quot;: &quot;com.android.calculator2&quot;,
    &quot;appActivity&quot;: &quot;com.android.calculator2.Calculator&quot;,
    &quot;appPackage&quot;: &quot;com.android.calculator2&quot;,
    &quot;platformVersion&quot;: &quot;8.0&quot;,
    &quot;automationFramework&quot;: &quot;Katalon&quot;,
    &quot;platformName&quot;: &quot;Android&quot;,
    &quot;deviceName&quot;: &quot;Nexus 6&quot;
  },
  &quot;capabilities&quot;: {
    &quot;firstMatch&quot;: [
      {
        &quot;appium:app&quot;: &quot;com.android.calculator2&quot;,
        &quot;appium:appActivity&quot;: &quot;com.android.calculator2.Calculator&quot;,
        &quot;appium:appPackage&quot;: &quot;com.android.calculator2&quot;,
        &quot;automationFramework&quot;: &quot;Katalon&quot;,
        &quot;appium:deviceName&quot;: &quot;Nexus 6&quot;,
        &quot;platformName&quot;: &quot;android&quot;,
        &quot;appium:platformVersion&quot;: &quot;8.0&quot;
      }
    ]
  }
}
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.extractDesiredCapability(WebDriverRequest.java:74)
	at org.openqa.grid.web.servlet.handler.SeleniumBasedRequest.&lt;init&gt;(SeleniumBasedRequest.java:102)
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.&lt;init&gt;(WebDriverRequest.java:36)
	at org.openqa.grid.web.servlet.handler.WebDriverRequestFactory.createFromRequest(WebDriverRequestFactory.java:30)
	at org.openqa.grid.web.servlet.handler.SeleniumBasedRequest.createFromRequest(SeleniumBasedRequest.java:73)
	at org.openqa.grid.web.servlet.DriverServlet.process(DriverServlet.java:83)
	... 33 more
Caused by: java.lang.IllegalArgumentException: Illegal key values seen in w3c capabilities: [automationFramework]
	at org.openqa.selenium.remote.NewSessionPayload.lambda$validate$5(NewSessionPayload.java:205)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:439)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:440)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:440)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485)
	at org.openqa.selenium.remote.NewSessionPayload.validate(NewSessionPayload.java:209)
	at org.openqa.selenium.remote.NewSessionPayload.&lt;init&gt;(NewSessionPayload.java:154)
	at org.openqa.selenium.remote.NewSessionPayload.create(NewSessionPayload.java:105)
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.extractDesiredCapability(WebDriverRequest.java:66)
	... 38 more
</pre>
<h3>Caused by:</h3><pre>org.openqa.grid.common.exception.GridException: Cannot extract a capabilities from the request: {
  &quot;desiredCapabilities&quot;: {
    &quot;app&quot;: &quot;com.android.calculator2&quot;,
    &quot;appActivity&quot;: &quot;com.android.calculator2.Calculator&quot;,
    &quot;appPackage&quot;: &quot;com.android.calculator2&quot;,
    &quot;platformVersion&quot;: &quot;8.0&quot;,
    &quot;automationFramework&quot;: &quot;Katalon&quot;,
    &quot;platformName&quot;: &quot;Android&quot;,
    &quot;deviceName&quot;: &quot;Nexus 6&quot;
  },
  &quot;capabilities&quot;: {
    &quot;firstMatch&quot;: [
      {
        &quot;appium:app&quot;: &quot;com.android.calculator2&quot;,
        &quot;appium:appActivity&quot;: &quot;com.android.calculator2.Calculator&quot;,
        &quot;appium:appPackage&quot;: &quot;com.android.calculator2&quot;,
        &quot;automationFramework&quot;: &quot;Katalon&quot;,
        &quot;appium:deviceName&quot;: &quot;Nexus 6&quot;,
        &quot;platformName&quot;: &quot;android&quot;,
        &quot;appium:platformVersion&quot;: &quot;8.0&quot;
      }
    ]
  }
}
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.extractDesiredCapability(WebDriverRequest.java:74)
	at org.openqa.grid.web.servlet.handler.SeleniumBasedRequest.&lt;init&gt;(SeleniumBasedRequest.java:102)
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.&lt;init&gt;(WebDriverRequest.java:36)
	at org.openqa.grid.web.servlet.handler.WebDriverRequestFactory.createFromRequest(WebDriverRequestFactory.java:30)
	at org.openqa.grid.web.servlet.handler.SeleniumBasedRequest.createFromRequest(SeleniumBasedRequest.java:73)
	at org.openqa.grid.web.servlet.DriverServlet.process(DriverServlet.java:83)
	at org.openqa.grid.web.servlet.DriverServlet.doPost(DriverServlet.java:69)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.seleniumhq.jetty9.servlet.ServletHolder.handle(ServletHolder.java:865)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doHandle(ServletHandler.java:535)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:146)
	at org.seleniumhq.jetty9.security.SecurityHandler.handle(SecurityHandler.java:548)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doScope(SessionHandler.java:1564)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.Server.handle(Server.java:503)
	at org.seleniumhq.jetty9.server.HttpChannel.handle(HttpChannel.java:364)
	at org.seleniumhq.jetty9.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.seleniumhq.jetty9.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.seleniumhq.jetty9.io.FillInterest.fillable(FillInterest.java:103)
	at org.seleniumhq.jetty9.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.seleniumhq.jetty9.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: Illegal key values seen in w3c capabilities: [automationFramework]
	at org.openqa.selenium.remote.NewSessionPayload.lambda$validate$5(NewSessionPayload.java:205)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:439)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:440)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:440)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485)
	at org.openqa.selenium.remote.NewSessionPayload.validate(NewSessionPayload.java:209)
	at org.openqa.selenium.remote.NewSessionPayload.&lt;init&gt;(NewSessionPayload.java:154)
	at org.openqa.selenium.remote.NewSessionPayload.create(NewSessionPayload.java:105)
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.extractDesiredCapability(WebDriverRequest.java:66)
	... 38 more
</pre>
<h3>Caused by:</h3><pre>java.lang.IllegalArgumentException: Illegal key values seen in w3c capabilities: [automationFramework]
	at org.openqa.selenium.remote.NewSessionPayload.lambda$validate$5(NewSessionPayload.java:205)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:439)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:440)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:440)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485)
	at org.openqa.selenium.remote.NewSessionPayload.validate(NewSessionPayload.java:209)
	at org.openqa.selenium.remote.NewSessionPayload.&lt;init&gt;(NewSessionPayload.java:154)
	at org.openqa.selenium.remote.NewSessionPayload.create(NewSessionPayload.java:105)
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.extractDesiredCapability(WebDriverRequest.java:66)
	at org.openqa.grid.web.servlet.handler.SeleniumBasedRequest.&lt;init&gt;(SeleniumBasedRequest.java:102)
	at org.openqa.grid.web.servlet.handler.WebDriverRequest.&lt;init&gt;(WebDriverRequest.java:36)
	at org.openqa.grid.web.servlet.handler.WebDriverRequestFactory.createFromRequest(WebDriverRequestFactory.java:30)
	at org.openqa.grid.web.servlet.handler.SeleniumBasedRequest.createFromRequest(SeleniumBasedRequest.java:73)
	at org.openqa.grid.web.servlet.DriverServlet.process(DriverServlet.java:83)
	at org.openqa.grid.web.servlet.DriverServlet.doPost(DriverServlet.java:69)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.seleniumhq.jetty9.servlet.ServletHolder.handle(ServletHolder.java:865)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doHandle(ServletHandler.java:535)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:146)
	at org.seleniumhq.jetty9.security.SecurityHandler.handle(SecurityHandler.java:548)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203)
	at org.seleniumhq.jetty9.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.seleniumhq.jetty9.server.session.SessionHandler.doScope(SessionHandler.java:1564)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201)
	at org.seleniumhq.jetty9.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.seleniumhq.jetty9.server.Server.handle(Server.java:503)
	at org.seleniumhq.jetty9.server.HttpChannel.handle(HttpChannel.java:364)
	at org.seleniumhq.jetty9.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.seleniumhq.jetty9.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.seleniumhq.jetty9.io.FillInterest.fillable(FillInterest.java:103)
	at org.seleniumhq.jetty9.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.seleniumhq.jetty9.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.seleniumhq.jetty9.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:748)
</pre>
<hr><a href="http://eclipse.org/jetty">Powered by Jetty:// 9.4.z-SNAPSHOT</a><hr/>
```  

Exception from Selenium Grid: 

![Selection_053](https://user-images.githubusercontent.com/46138753/63154727-ededa280-c019-11e9-9197-2a37e668a6db.png)

Tried run configurations: 

![Selection_056](https://user-images.githubusercontent.com/46138753/63155057-8dab3080-c01a-11e9-9158-b8fcd2a39885.png)

![Selection_055](https://user-images.githubusercontent.com/46138753/63155073-9439a800-c01a-11e9-9aa6-523928df28d2.png)

**Comment by duyluonganh** [comment]
Hi @DmytroLapshyn, @maria-zaitseva 

We will update this fix in v7.0.0 release.

Thanks for your report
